query_execution/CMakeLists.txt

```diff
@@ -82,12 +86,8 @@ target_link_libraries(quickstep_queryexecution_Foreman
                       quickstep_threading_ThreadUtil
                       quickstep_utility_EqualsAnyConstant
                       quickstep_utility_Macros
-                      tmb)
-target_link_libraries(quickstep_queryexecution_ForemanLite
-                      glog
-                      quickstep_threading_Thread
-                      quickstep_utility_Macros
-                      tmb)
+                      tmb
+                      ${GFLAGS_LIB_NAME})
```

Comment:
No, it starts w/ `$`, so it should be in the last.

See [here](https://github.com/apache/incubator-quickstep/blob/master/storage/CMakeLists.txt#L291) for example regarding `${PROTOBUF_LIBRARY}`.